en/django_forms/images/file-earmark-plus.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-file-earmark-plus" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think this file is used anywhere, right?

Reply:
You are correct. It was initially added to provide download link and found a bug. So changed link to an absolute url from GitHub. I'll remove it.

Reply:
Resolved in https://github.com/DjangoGirls/tutorial/pull/1681/commits/a8b94e64223709fdddf5c36bf9510722cf764155